lib/is-semantic-message.js

```diff
@@ -1,7 +1,10 @@
 const commitTypes = Object.keys(require('conventional-commit-types').types)
 const { validate } = require('parse-commit-message')
 
-module.exports = function isSemanticMessage (message, scopes) {
+module.exports = function isSemanticMessage (message, scopes, allowedMergeCommits) {
```

Comment:
```suggestion
module.exports = function isSemanticMessage (message, scopes, alloweMergeCommits) {
```

Reply:
I am not native English, we can use allow or allowed here but not allowe because is not an existing word right?

Reply:
Ha! Right. Fixed. 😅